pymongo/synchronous/monitor.py

```diff
@@ -302,6 +324,19 @@ def _check_once(self) -> ServerDescription:
                 self._listeners.publish_server_heartbeat_succeeded(
                     address, round_trip_time, response, response.awaitable
                 )
+            if _SDAM_LOGGER.isEnabledFor(logging.DEBUG):
+                _debug_log(
+                    _SDAM_LOGGER,
+                    topologyId=self._topology._topology_id,
+                    driverConnectionId=conn.id,
+                    serverConnectionId=conn.server_connection_id,
+                    serverHost=address[0],
+                    serverPort=address[1],
+                    awaited=awaited,
+                    durationMS=round_trip_time,
```

Comment:
Here and elsewhere: round_trip_time is a float in seconds, but durationMS should be in milliseconds.

Reply:
Still need to make this change in the other locations where durationMS is passed.